test/runtests.jl

```diff
@@ -1,7 +1,5 @@
 using Oscar
 using Test
-using Aqua
-using Documenter
```

Comment:
Could you additionally remove `Documenter` from `test/Project.toml`?

Reply:
Done. Thanks!